Assets/Scripts/Player/PlayerController.cs

```diff
@@ -345,4 +385,38 @@ public class PlayerController : MonoBehaviour {
 		doDash = false;
 		animator.SetBool(isDashingHash, false);
 	}
+
+	private void StartCrouch() {
+		isCrouching = true;
+		isCrouched = true;
+		crouchTransitionTimer = 0f;
+		onCrouch.Invoke();
+	}
+
+	private void Uncrouch() {
+		isCrouching = true;
+		isCrouched = false;
+		crouchTransitionTimer = 0f;
+		onUncrouch.Invoke();
+	}
+
+	private void UpdateCrouchColliders(BoxCollider2D from, BoxCollider2D to, BoxCollider2D gFrom, BoxCollider2D gTo) {
```

Comment:
The 'g' prefix does not help in understanding what the parameter is for. Just call them `groundFrom` and to.